Source/Core/DolphinWX/Main.cpp

```diff
@@ -96,21 +96,72 @@ bool DolphinApp::Initialize(int& c, wxChar **v)
 
 bool DolphinApp::OnInit()
 {
+	if (!wxApp::OnInit() || DolphinEmulatorDotComTextFileExists())
```

Comment:
Can we rename `DolphinEmulatorDotComTextFileExists` to something more generic, in case we want to add stuff to it in the future? Also maybe move it to `UICommon` so DolphinQt can use it too.